client/my-sites/comments/comment/comment-author-more-info.jsx

```diff
@@ -0,0 +1,35 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import InfoPopover from 'components/info-popover';
+
+export class CommentAuthorMoreInfo extends Component {
+	storeLabelRef = label => ( this.authorMoreInfoLabel = label );
+
+	storePopoverRef = popover => ( this.authorMoreInfoPopover = popover );
+
+	openAuthorMoreInfoPopover = event => this.authorMoreInfoPopover.handleClick( event );
+
+	render() {
+		return (
+			<div
+				className="comment__author-more-info"
+				onClick={ this.openAuthorMoreInfoPopover }
+				ref={ this.storeLabelRef }
+			>
+				<InfoPopover ignoreContext={ this.authorMoreInfoLabel } ref={ this.storePopoverRef }>
+					Author More Info
+				</InfoPopover>
+				<label>User Info</label>
```

Comment:
Don't forget to update these later :)

Reply:
Yes, of course! :)
I just wanted to show a popover there, but the actual user info requires so many selectors and actions that I'd have doubled this PR LOC 😄